content/bluetooth.md

```diff
@@ -72,44 +99,135 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
+Remove the device, allowing for a fresh, new connection.
+
+![Remove device](/images/bluetooth/bluetooth_3.png)
+
+Click search and pair the device again:
+
+![Pair device](/images/bluetooth/bluetooth_4.png)
 
 If `tlp` is installed, then there may be settings interfering with Bluetooth functionality.  Edit this file and disable Wifi and Bluetooth power saving features:
 
 ```bash
-sudo gedit /etc/default/tlp
+sudo nano /etc/tlp.conf
 ```
 
 ### Useful Programs
 
-There is a program called <u>Bluetooth Manager</u> which is included with <u>XFCE</u>. It can sometimes pair and trust Bluetooth devices better than the default <u>Bluetooth</u> settings. Install it with:
+There is a program called <u>Bluetooth Manager</u>. It can sometimes pair and trust Bluetooth devices better than the default <u>Bluetooth</u> settings. Install it with:
 
 ```bash
 sudo apt install blueman
 ```
 
+Start the bluetooth using systemctl:
+
+```bash
+sudo systemctl start bluetooth
+```
+
+```bash
+sudo systemctl enable bluetooth
+```
+
+Check the status of the bluetooth
+
+```bash
+sudo systemctl status bluetooth
+```
+
+Sample output:
+
+![bluetooth status systemd](/images/bluetooth/bluetooth_5.png)
+
 Then, run <u>Bluetooth Manager</u>. Check for the device being trusted, and also try re-pairing in that program.
 
+### Using bluetoothctl
+
+Using bluetoothctl over a UI offers more control, flexibility, and efficiency, especially for advanced users. It allows precise management of Bluetooth devices via the terminal, which is faster than navigating through graphical menus. For troubleshooting, bluetoothctl provides direct feedback and logs, which can help identify connection issues, detect devices, or configure settings in real time. It’s also useful for headless or remote setups where a UI might not be available.
+
+To get started, ensure Bluetooth is unblocked by running rfkill to check and enable it if necessary. Use the command:
+
+```bash
+rfkill unblock bluetooth
+```
+to ensure that Bluetooth is not disabled at the system level.
+
+Type:
+```
+bluetoothctl
+```
+![bluetoothclt](/images/bluetooth/bluetooth_6.png)
+
+If you have multiple Bluetooth controllers, choose the one you wish to connect to the device:
+
+Check list of controllers:
+```
+list
+```
+
+Check controller information:
+```
+info <controller_address>
+```
+
+Select the controller you want to use:
+```
+select <mac address>
+```
+
+Make sure to power it on
+```
+power on
+```
+![bluetoothctl power on](/images/bluetooth/bluetooth_7.png)
+
+Look for the device you want to connect:
+```
+scan on
+```
+Add trusted device
+```
+trust <mac address>
+```
+See list of paired devices:
+
+```
+devices
+```
+
+To connect the device:
+```
+connect <mac address>
+```
+![bluetootctl list scan trust connect](/images/bluetooth/bluetooth_8.png)
+
+Successful device connection:
+
+![bluetooothctl device connected](/images/bluetooth/bluetooth_9.png)
+
 ### Useful Commands
```

Comment:
Can 'useful commands' section go to the beginning of the article after the how to remove and pair instructions?